### ISSUE TITLE
In Registration class the javadocs say withEmail instead of withUserAttributes

### ISSUE THREAD

**Issue by bryant1410** [intercom/intercom-android#238]
## Version info
  - Intercom for Android version: 3..0.2

The example says:

```java
Registration.create()
    .withUserId("12345")
    .withEmail("test@email.com")
    .withEmail(userAttributes);
```

But it should say:

```java
Registration.create()
    .withUserId("12345")
    .withEmail("test@email.com")
    .withUserAttributes(userAttributes);
```

**Comment by ronocod** [comment]
Thanks, this will be fixed in the next release.

**Comment by ronocod** [comment]
Fixed in 3.0.4.
